### PR TITLE
[zh-cn] sync task-tutorial-prereqs

### PR DESCRIPTION
**Pull request by asa3311**
sync task-tutorial-prereqs

### CODE REVIEW

File: content/zh-cn/includes/task-tutorial-prereqs.md [hunk 1]
```diff
@@ -11,4 +11,4 @@ or you can use one of these Kubernetes playgrounds:
 -->
 
 * [Killercoda](https://killercoda.com/playgrounds/scenario/kubernetes)
-* [玩转 Kubernetes](http://labs.play-with-k8s.com/)
+* [玩转 Kubernetes](https://labs.play-with-k8s.com/)
```
Comment (windsonsea) [inline content/zh-cn/includes/task-tutorial-prereqs.md:14]: Do we need to copy and hide the corresponding en text by following our [localization-zh style guide](https://kubernetes.io/zh-cn/docs/contribute/localization_zh/#commented-en-text)?

File: content/zh-cn/includes/task-tutorial-prereqs.md [hunk 2]
```diff
@@ -1,14 +1,18 @@
...
-如果你还没有集群，你可以通过 [Minikube](https://minikube.sigs.k8s.io/docs/tutorials/multi_node/) 
-构建一个你自己的集群，或者你可以使用下面任意一个 Kubernetes 工具构建：
 <!--
 You need to have a Kubernetes cluster, and the kubectl command-line tool must
 be configured to communicate with your cluster. It is recommended to run this tutorial on a cluster with at least two nodes that are not acting as control plane hosts. If you do not already have a
 cluster, you can create one by using
 [minikube](https://minikube.sigs.k8s.io/docs/tutorials/multi_node/)
 or you can use one of these Kubernetes playgrounds:
 -->
+你必须拥有一个 Kubernetes 的集群，同时你的 Kubernetes 集群必须带有 kubectl 命令行工具。
```
Comment (tengqm) [inline content/zh-cn/includes/task-tutorial-prereqs.md:8]:
```suggestion
你必须拥有一个 Kubernetes 的集群，同时你必须配置 kubectl 命令行工具与你的集群通信。
```

File: content/zh-cn/includes/task-tutorial-prereqs.md [hunk 3]
```diff
@@ -1,14 +1,18 @@
...
+建议在至少有两个节点的集群上运行本教程，且这些节点不作为控制平面主机。
```
Comment (tengqm) [inline content/zh-cn/includes/task-tutorial-prereqs.md:9]: 英文没读懂么？

```suggestion
建议在至少有两个不作为控制平面主机的节点的集群上运行本教程。
```

File: content/zh-cn/includes/task-tutorial-prereqs.md [hunk 4]
```diff
@@ -1,14 +1,18 @@
...
+如果你还没有集群，你可以通过 [Minikube](https://minikube.sigs.k8s.io/docs/tutorials/multi_node/) 
+构建一个你自己的集群，或者你可以使用下面任意一个 Kubernetes 工具构建：
```
Comment (tengqm) [inline content/zh-cn/includes/task-tutorial-prereqs.md:11]:
```suggestion
构建一个你自己的集群，或者你可以使用下面的 Kubernetes 练习环境之一：
```

File: content/zh-cn/includes/task-tutorial-prereqs.md (hunk 3 above)
Comment (asa3311) [reply content/zh-cn/includes/task-tutorial-prereqs.md]: 不好意思，我只是调整了下格式，没关注以前的译文是否合理，已修改，请审阅

**Pull request merged**